biolink-model.yaml

```diff
@@ -2927,7 +2948,7 @@ classes:
       - DOID
       - OMIM
       - ORPHANET
-      - ORPHA
```

Comment:
if nothing else 
it was a prefix chosen by the authors of the Dipper ingest before my time 
for what is now "ORPHA'